templates/kubernetes/onpremises/hosts.yaml.tpl

```
@@ -156,6 +156,10 @@ all:
         kubernetes_apiserver_certSANs:
 {{ .spec.kubernetes.advanced.apiServerCertSANs | toYaml | indent 10 }}
         {{- end }}
+        {{- if and (hasKeyAny .spec.kubernetes "advanced") (hasKeyAny .spec.kubernetes.advanced "kubeProxy") (not (index .spec.kubernetes.advanced.kubeProxy "enabled")) }}
+        kubeadm_skip_phases:
+          - "addon/kube-proxy"
+        {{- end }}
```

Same here:
```suggestion
        {{ template "IncludeIfKubeProxyEnabled" (dict "state" false "config" . "object" ("kubeadm_skip_phases: [\"addon/kube-proxy\"]") }}
```

(double check this suggestion, maybe there's a better way to properly create this array)